### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-python-library.git`](https://github.com/cisagov/skeleton-python-library.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/gophish-tools.git gophish-tools
    cd gophish-tools
    git remote add skeleton https://github.com/cisagov/skeleton-python-library.git
    git remote set-url --push skeleton no_push
    git switch develop
    git switch --create lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/dependabot.yml .github/workflows/build.yml src/example/__init__.py src/templates/_version.py tests/conftest.py 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Confirm each item in the "Pre-approval checklist" below.

1. Remove any of the checklist items that do not apply.

1. Ensure every remaining checkbox has been checked.

1. Mark this draft pull request "Ready for review".

## ✅ Pre-approval checklist ##

- [x] ✌️ The conflicts in this pull request have been resolved.
- [x] All relevant type-of-change labels have been added.
- [x] All new and existing tests pass.

---

> [!NOTE]
> You are seeing this because one of this repository's maintainers has
> configured [Lineage] to open pull requests.

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by dv4harr10 — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -106,37 +121,55 @@ jobs:
...
+           --single-branch \
+           $TERRAFORM_DOCS_REPO_URL /tmp/terraform-docs
+      - name: Build and install terraform-docs binary
+        run: |
+          go build \
+          -C /tmp/terraform-docs \
+          -o $(go env GOPATH)/bin/terraform-docs
       - name: Install dependencies
+        # We don't upgrade setuptools here because the versions of
+        # gophish on PYPI are ancient and depend on an equally ancient
```
Comment (mcdonnnj) [inline .github/workflows/build.yml]:
```suggestion
        # gophish on PyPI are ancient and depend on an equally ancient
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -380,15 +417,21 @@ jobs:
...
         with:
           name: dist-${{ matrix.python-version }}
           path: dist
       - id: find-wheel
         name: Get the name of the retrieved wheel (there should only be one)
         run: echo "wheel=$(ls dist/*whl)" >> $GITHUB_OUTPUT
       - name: Update core Python packages
-        run: python -m pip install --upgrade pip setuptools wheel
+        # We don't upgrade setuptools here because the versions of
+        # gophish on PYPI are ancient and depend on an equally ancient
```
Comment (mcdonnnj) [inline .github/workflows/build.yml]:
```suggestion
        # gophish on PyPI are ancient and depend on an equally ancient
```

File: src/_version.py
Comment (mcdonnnj) [inline src/_version.py:1]: Do we want to bump this version?

**Review by mcdonnnj — APPROVED** [review]
A few minor items of feedback. Cursed but necessary workarounds. If this package is in regular use we may honestly want to consider forking the [upstream API project](https://github.com/gophish/api-client-python) just so dependencies can be updated. Looking at the issues in the project this appears to be a longstanding problem with no interaction from maintainers.

File: src/_version.py
Comment (jsf9k) [reply src/_version.py:1]: No code has really changed here, and Python 3.6 was already unsupported, so I didn't think a version bump was warranted.

**Review by dav3r — APPROVED** [review]

File: src/_version.py
Comment (mcdonnnj) [reply src/_version.py:1]: Ahh I missed that Python 3.6 was already unsupported.

**Pull request merged**